python_ta/debug/snapshot.py

```diff
@@ -40,6 +41,7 @@ def snapshot(
     save: bool = False,
     memory_viz_args: Optional[list[str]] = None,
     memory_viz_version: str = "latest",
+    include: Optional[tuple[str, ...]] = (".*",),
```

Comment:
Make the default `None`, and in the code below if `include is None` then skip the checks.

Make the inner type annotation an `Iterable[str]` (import from `typing`) to allow different kinds of collections to be passed in.

Also, just to be complete, the contained type can be `str | re.Pattern`, as either of those types are permitted by `re.compile`.